doorbell/doorbell_backend/src/logupdate.py

```diff
@@ -1,31 +1,81 @@
-import os
-from datetime import datetime
 import csv
-import time
-import re
-from pathlib import Path
+from datetime import datetime
+from time import ctime
+from re import search, sub
+from os.path import isfile, join
+from json import load, dump
+
+import connect
+
+import constants
+
+
+def updateStatus(update_status):
+    with open(constants.CONNECT_SETTINGS_PATH, "r") as jsonFile:
+        data = load(jsonFile)
+        if (
+            data["Connection Settings"][3]["Connection Status"] == "False"
```

Comment:
Is there a better way to do this than using True/False strings? 0/1 maybe

Reply:
0 and 1's work, but True/False strings are more readable if anyone ever needed to manually edit the json. The computational overhead is minimal.

Reply:
0/1 would work, but it would make editing the json file harder if the user ever needed to manually change something. It depends on what the frontend can write to the json file. The computational difference between a string comparison and int comparison probably don't matter too much.